### PR TITLE
Add forms

### PR DESCRIPTION
**Pull request by omar-muhamad**
In this PR I did:

- [ ] [Add post form](https://github.com/Omar-Muhamad/Rails-Blog-App/commit/39199c2190dbb7f9b307c84b7504065a7f85c4b0).
- [ ] [Add comment form](https://github.com/Omar-Muhamad/Rails-Blog-App/commit/20dc6dfbae5b6987751394b46abf7a72b7c9c201).
- [ ] [Add like to posts show view](https://github.com/Omar-Muhamad/Rails-Blog-App/commit/4fbe157a6f546bcd1fc095138bf3d807676cd750).

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class CommentsController < ApplicationController
```
Comment (rloterh) [inline app/controllers/comments_controller.rb:1]: - [x] The _new_ action is missing in the _comments controller_! Kindly ensure that the _comments controller_ have both _new_ and _create_ actions similar to the one shown below:

```ruby
# controllers/comments_controller.rb

class CommentsController < ApplicationController
  ...

  def new
    @comment = Comment.new
  end

	def create
    @comment = @current_user.comments.new(params)
    @comment.post_id = params[:post_id]

    if @comment.save
      redirect_to post_path(@comment.post)
    else
      @article = Post.find(params[:post_id])
      redirect_to post_path(@post)
    end
  end
 
	...
end
```

File: config/routes.rb [hunk 2]
```diff
@@ -1,9 +1,12 @@
 # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 Rails.application.routes.draw do
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show, :new]
   end
-
+  post '/users/:user_id/posts', to: 'posts#create', as: 'create_user_post'
+  post '/users/:user_id/posts/:id', to: 'comments#create', as: 'create_post_comment'
```
Comment (rloterh) [inline config/routes.rb]: - [x] There should be a **pair** of routes in the _config/routes.rb_ file for creating comments. Kindly improve upon this aspect similar to the one shown below:

```ruby
   POST   /posts/:post_id/comments(.:format)     comments#create
   GET   /posts/:post_id/comments/new(.:format)     comments#new
```

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Omar, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Good Points 👍  
- Good ReadMe
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rukundoeric — APPROVED** [review]
Hi @Omar-Muhamad,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------

**Pull request merged**